### DISCUSSION TITLE
关于科研过滤器的试验品募集

### DISCUSSION THREAD

**Discussion by rekaime** [LmeSzinc/AzurLaneAutoScript#3383] [Q&A] [answered]
试验品募集有分紫色与蓝色品质，时长都是2h，如果我希望优先做紫色品质的，那么应该怎么编辑过滤器呢？
是不是说写上 `S4-E2` ，Alas就会优先选紫色的？然后如果我希望的顺序是 `试验品紫 > 资金4 > 试验品蓝` ，又该怎么做呢？
刚刚把先前的讨论大概看了一遍，似乎没发现类似问题。

**Comment by LmeSzinc** [comment, marked as answer]
`S4-E-315`, `S4-E-031` 算是文档没写的隐藏机制，毕竟没人愿意关心项目的编号

**Reply by rekaime** [reply]
ok，感谢解答（比心）
